build.gradle

```diff
@@ -4,11 +4,11 @@ plugins {
 }
 
 group = "org.jenkins-ci.plugins"
-description = "Jucies update center plugin"
+description = "Plugin adds additional Jucies update center that provides additional plugins distributed in more flexible way."
```

Comment:
1) to many "additional" :)
2) IMO something like "that allows you to install plugins from GitHub" will be better (since this description is for users, they don't want to know about "flexible ways" )